### PR TITLE
Landing page DO NOT MERGE

### PR DESCRIPTION
**Pull request by saracarl**

### CODE REVIEW

File: app/views/shared/_landingheader.erb [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```
Comment (benwbrum) [inline app/views/shared/_landingheader.erb:4]: This file seems to contain everything that also appears in _landingbodyheader.html.erb.  Are we duplicating the header stuff?  Is this perhaps dead code?

File: app/views/shared/_landingfooter.erb [hunk 2]
```diff
@@ -0,0 +1,81 @@
+    <!-- Footer -->
```
Comment (benwbrum) [inline app/views/shared/_landingfooter.erb:1]: I don't understand why this isn't a `.html.erb` file.  I suspect Rails will render it as HTML, but ???

File: app/views/static/digital_scholarship.html.erb [hunk 3]
```diff
@@ -0,0 +1,734 @@
+<%= render "shared/landingheader" %>
+<body>
+<%= render "shared/landingbodyheader" %>
```
Comment (benwbrum) [inline app/views/static/digital_scholarship.html.erb:3]: Indent.

File: app/views/static/digital_scholarship.html.erb [hunk 4]
```diff
@@ -0,0 +1,734 @@
...
+                                        <span>Scholar of Medieval Francophonie and Italian Historiographies</span>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="box">
+                            <div class="item">
+                                <p><img src="assets/icons/quote01.svg" alt="" class="top">We have an enormous amount of understudied documentation that could yield meaningful findings, if people could just get to the stuff — FromThePage helps make access possible.
+<br>
```
Comment (benwbrum) [inline app/views/static/digital_scholarship.html.erb:638]: Indent this (and similar second/third lines) under the `p`

File: app/views/static/digital_scholarship.html.erb [hunk 5]
```diff
@@ -0,0 +1,734 @@
...
+                        </div>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- Crowd Testimonials End -->
+
+<%= render "shared/landingfooter" %>
```
Comment (benwbrum) [inline app/views/static/digital_scholarship.html.erb:730]: indent

File: app/views/static/landing_page.html.erb [hunk 6]
```diff
@@ -0,0 +1,613 @@
+<%= render "shared/landingheader" %>
+<body>
+<%= render "shared/landingbodyheader" %>
```
Comment (benwbrum) [inline app/views/static/landing_page.html.erb:3]: indent this and footer, plus any paragraph continuations

**Review by benwbrum — CHANGES_REQUESTED** [review]
Minor indentation issues; possible dead code in head file and dead files under `flags`

File: app/views/shared/_landingheader.erb (hunk 1 above)
Comment (saracarl) [reply app/views/shared/_landingheader.erb:4]: Please take a closer look at landingbodyheader -- different content.   Here's the first 3 lines:
```<!-- Header -->
    <header>
        <div class="container">
            <a href="index.html" class="logo">```

File: app/views/shared/_landingfooter.erb (hunk 2 above)
Comment (saracarl) [reply app/views/shared/_landingfooter.erb:1]: fixed.

File: app/views/static/digital_scholarship.html.erb (hunk 3 above)
Comment (saracarl) [reply app/views/static/digital_scholarship.html.erb]: fixed.

File: app/views/static/digital_scholarship.html.erb (hunk 4 above)
Comment (saracarl) [reply app/views/static/digital_scholarship.html.erb]: fixed.

File: app/views/static/digital_scholarship.html.erb (hunk 5 above)
Comment (saracarl) [reply app/views/static/digital_scholarship.html.erb]: fixed.

File: app/views/static/landing_page.html.erb (hunk 6 above)
Comment (saracarl) [reply app/views/static/landing_page.html.erb]: fixed.

File: app/views/shared/_landingheader.erb (hunk 1 above)
Comment (saracarl) [reply app/views/shared/_landingheader.erb]: fixed.

**Review by benwbrum — APPROVED** [review]

**Pull request merged**